src/samples/flow/netctl/network-connect.fbp

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env sol-fbp-runner
+
+# This file is part of the Soletta (TM) Project
+#
+# Copyright (C) 2015 Intel Corporation. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This program will print to console a list of all network services
+#
+# Examples:
+#  * connect or disconnect network services
+#    $ ./network-connect.fbp Guest
+
+name(app/argv:index=1)
+error-msg(converter/empty-to-string:output_value="Network service name is mandatory")
+Error(console)
+quit(app/quit)
+service(netctl/service)
+tick(constant/empty)
+timer(timer:interval=3000)
+btoe(converter/boolean-to-empty)
+etoi(converter/empty-to-int:output_value=0)
+
+name ERROR -> IN error-msg OUT -> IN Error
+service ERROR -> ERROR quit
+
+name OUT -> NAME service
+tick OUT -> CONNECT service OUT -> IN _(console:prefix="state: ")
+timer OUT -> TRIGGER service ONLINE -> PULSE_IF_TRUE btoe OUT -> IN etoi OUT -> CODE quit
```

Comment:
why do you need this `timer OUT -> TRIGGER service`?  You can drop it and `timer(timer:interval=3000)`.